src/client/src/Domains/Messages/ConversationPreview/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import Card from '@material-ui/core/Card';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+
+import { IUser } from 'Domains/Accounts/api';
+import { IConversation } from 'Domains/Messages/api';
+
+interface Props {
+    conversation: IConversation;
+    onClick: (user: IUser) => void;
+    isSelected: boolean;
+}
+
+const useStyles = makeStyles((theme: Theme) =>
+    createStyles({
+        NonSelected: {
+            '&:hover': { backgroundColor: '#efefef' },
+        },
+        Selected: {
+            backgroundColor: '#efefef',
+            borderColor: theme.palette.primary.main,
+        },
+    })
+);
+
+// Convert dateString to a readable formatting.
+const getDateString = (dateString: string): string => {
```

Comment:
I'm using exactly the same function in MessageDialog. Is there any way to write this function in one place and call from both ConversationPreview/index.tsx and MessageDialog/index.tsx?

Reply:
Done.